### ISSUE TITLE
Issue with Django 1.11 and DAL 3.2.7, Jquery is not loaded properly

### ISSUE THREAD

**Issue by maxcmoi89** [yourlabs/django-autocomplete-light#858]
I cannot seem to have dal 3.2.7 work with django 1.11. There is an issue with Jquery, when the page tries to load Select2 in the admin it does not find the jquery library.

Screenshot of the error given by select 2:
![screenshot from 2017-06-04 12-14-55](https://cloud.githubusercontent.com/assets/8608425/26760700/9c266e46-491f-11e7-8776-07a210f06760.png)
Screenshot of script tags of the django admin page.
![screenshot from 2017-06-04 12-14-36](https://cloud.githubusercontent.com/assets/8608425/26760701/9c28d1fe-491f-11e7-9e91-b19e7e43dca7.png)

Note that I did include `dal` and `dal_select2` before `django.contrib.admin` in the installed apps.

**Comment by maxcmoi89** [comment]
Ok, I know what the problem is:

I tried upgrading to dal 3.2.8 and it did not work, I tried to downgrade to django 1.10 and it still did not work.

*BUT* when I tried with another browser (Firefox) it worked like a charm, same with chrome incognito window. I concluded chrome probably hadn't refreshed it's cache or something. I refreshed the page with `ctrl + SHIFT + R` and everything seems to be working fine now.

TL; DR; 
It was not a problem with DAL but a problem with the google chrome cache.
